pkg/netpol/diff/diff.go

```diff
@@ -215,7 +214,7 @@ func (c *ConnsPair) isSrcOrDstPeerIPType(checkSrc bool) bool {
 const ingressPodName = "ingress-controller"
 
 func isIngressControllerPeer(peer eval.Peer) bool {
-	return strings.Contains(peer.String(), ingressPodName) // peer.String() == "{"+ingressPodName+"}"
+	return peer.String() == "{"+ingressPodName+"}"
```

Comment:
can change as follows:

```
const ingressPodName = "{ingress-controller}"

func isIngressControllerPeer(peer eval.Peer) bool {
	return peer.String() == ingressPodName
```

Reply:
also, rename `ingressPodName` to `ingressControllerPodName`

Reply:
done